securedrop/translations/ro/LC_MESSAGES/messages.po

```diff
@@ -983,6 +983,16 @@ msgstr ""
 msgid "Forgot your codename?"
 msgstr "Ți-ai uitat numele de cod?"
 
+#: source_templates/footer.html:7
```

Comment:
Let's make sure we test the footer behavior with `ro` and ensure the english-language messages are properly displayed for these strings.

Reply:
They are.

![romanian](https://user-images.githubusercontent.com/47430598/64989685-06144400-d89c-11e9-8d7f-998f72be90a3.png)